src/storage/BaseProcessor.h

```diff
@@ -98,6 +98,17 @@ class BaseProcessor {
         }
     }
 
+    void handleLeaderChanged(GraphSpaceID spaceId, PartitionID partId) {
+        auto addrRet = kvstore_->partLeader(spaceId, partId);
+        if (ok(addrRet)) {
+            auto leader = value(std::move(addrRet));
+            this->pushResultCode(cpp2::ErrorCode::E_LEADER_CHANGED, partId, leader);
+        } else {
+            LOG(ERROR) << "Fail to get part leader, spaceId: " << spaceId
```

Comment:
Be careful.  If we can't get the leader information (which is normal),  we should return the empty leader information and errorcode Leader_change. 

By the way,  could we unify the duplicate code about leader change logic with handleAsync in BaseProcessor

Reply:
Oh my fault